mmv1/templates/terraform/encoders/dns_resource_record_set.go.erb

```diff
@@ -0,0 +1,18 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2020 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+
+// The only acceptable kind right now
+obj["kind"] = "dns#resourceRecordSet"
```

Comment:
Does the API require us to send this? We can eliminate it if not.

Reply:
API doesn't required it, hence elminated